tests/page-objects/contacts/admin-user.wdio.page.js

```diff
@@ -0,0 +1,68 @@
+const addUserButton = () => $('a#add-user');
+const addUserDialog = () => $('h2=Add user');
+const userName = () => $('#edit-username');
+const userFullName = () => $('#fullname');
+const userRole = () => $('#role');
+const userPlace = () => $('//span[@aria-labelledby="select2-facilitySelect-container"]');
+const userAssociatedContact = () => $('//span[@aria-labelledby="select2-contactSelect-container"]');
+const userPassword = () => $('#edit-password');
+const userConfirmPassword = () => $('#edit-password-confirm');
+const saveUserButton = () => $('a=Add user');
+const logoutButton = () => $('span=Log out');
```

Comment:
Same here, using text is questionable and looking at the html there isn't a great selector. Let's jump on a call and add one together.

Reply:
* As mentioned above, added the `id` attribute for the modal submit button
* For log out button there is actually a selector we could use: font awesome's log out icon, using that now